README.md

```diff
@@ -1,134 +1,104 @@
 # YouChoose.AI
 
-There are three components that compose this project.
+There are two components that compose this project.
 
 1. The browser extension. the core product of YouChoose.AI, sponsored by Ledger project round 3
 2. The backend of https://youtube.tracking.exposed, free software developed in 2018 under the ERC project ALEX (https://algorithms.exposed). This package since 2019 is maintained as an independent mechanism for Youtube algorithm accountability and enhanced to be the backend supporting YouChoose.
```

Comment:
We can add a link to the `yttrex` repo here

Reply:
yes